basic/exercise11/scriptSpec.js

```diff
@@ -1,15 +1,25 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals convertTemperatureFromCelsiusToFahrenheit*/
+/*globals convertTemperaturesFromFahrenheitToCelsius*/
 
-describe("Checking if  degrees Celsius ", function(){
-    it("is degrees Fahrenheit", function(){
-        var temperatureInDegreesFahrenheit =  convertTemperaturesFromCelsiusToFahrenheit(degreesCelsius);
-        expect(temperatureInDegreesFahrenheit).toBeTruthy();
+describe("Convert temperature:", function () {
+    it("0 degrees Celsius is 32 degrees Fahrenheit", function () {
+        var degreesFahrenheit = convertTemperatureFromCelsiusToFahrenheit(0);
+        expect(degreesFahrenheit).toBe(32);
+    });
+    it("10 degrees Celsius is 50 degrees Fahrenheit", function () {
+        var degreesFahrenheit = convertTemperatureFromCelsiusToFahrenheit(10);
+        expect(degreesFahrenheit).toBe(50);
     });
 });
 
-describe("Checking if  degrees Fahrenheit ", function(){
-    it("is degrees Celsius", function(){
-        var temperatureInDegreesCelsius =  convertTemperaturesFromFahrenheitToCelsius(degreesFahrenheit);
-        expect(temperatureInDegreesCelsius).toBeTruthy();
+describe("Convert temperature:", function () {
+    it("0 degrees Fahrenheit is -17.777777778 degrees Celsius.", function () {
+        var degreesCelsius = convertTemperaturesFromFahrenheitToCelsius(0);
+        expect(degreesCelsius).toEqual(-17.777777778);
+    });
+    it("60 degrees Fahrenheit is 15.555555556 degrees Celsius.", function () {
+        var degreesCelsius = convertTemperaturesFromFahrenheitToCelsius(60);
+        expect(degreesCelsius).toEqual(15.555555556);
```

Comment:
toBe or not toBe :)?
I mean toBe or toEqual? Because once you use toBe and in different place toEqual.